api/envoy/extensions/filters/http/credential_injector/v3/credential_injector.proto

```diff
@@ -14,29 +14,27 @@ option java_outer_classname = "CredentialInjectorProto";
 option java_multiple_files = true;
 option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/credential_injector/v3;credential_injectorv3";
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
-option (xds.annotations.v3.file_status).work_in_progress = true;
```

Comment:
High-level suggestion:
I suggest keeping it WIP, until you run this in prod for a short period of time, and then decide whether the API should be finalized or not.